handlers/dimension/utils.go

```diff
@@ -0,0 +1,60 @@
+package dimension
+
+import (
+	"errors"
+	"github.com/ONSdigital/dp-dd-api-stub/stub"
+	"github.com/ONSdigital/dp-dd-api-stub/models"
+	"fmt"
+	"os"
+	"encoding/json"
+)
+
+func findDimension(datasetID string, dimensionID string)  (dimension *models.Dimension, err error) {
+	raw, err := stub.Asset("data/datasets/" + datasetID + "/dimensions.json")
+	if err != nil {
+		fmt.Println(err.Error())
+		os.Exit(1)
+	}
+
+	var dimensions []models.Dimension = []models.Dimension{}
+	json.Unmarshal(raw, &dimensions)
+
+	for _, dimension := range dimensions {
+		if dimension.ID == dimensionID {
+			return &dimension, nil
+		}
+	}
+	return &models.Dimension{}, errors.New("Dimension id " + dimensionID + " not found.")
```

Comment:
Could we do it as a part of another refactoring task please. There is plenty of places we are using it that way so it kind of falls outside of the main aim of current pull request. I am pretty much reusing that bid of code from earlier commit. 
Merging current PR would keep FE moving forward and then we can discuss that problem as a separate refactoring task.